appinventor/components-ios/src/CircularProgress.swift

```diff
@@ -0,0 +1,101 @@
+// -*- mode: swift; swift-mode:basic-offset: 2; -*-
+// Copyright 2017-2023 MIT, All rights reserved
```

Comment:
Since this is new to App Inventor this year, you only have to use 2023.

Reply:
@ewpatton updated to 2023